prow/config/presubmits.yaml

```diff
@@ -1245,6 +1245,55 @@ presubmits:
       - emptyDir: {}
         name: varlibdocker
 
+  - name: istio-push-images-2.3
+    trigger: (?m)^/test( | .* )push\-images,?($|\s.*)
+    rerun_command: /test push-images
+    skip_report: false
+    max_concurrency: 2
+    always_run: true
+    branches:
+    - ^maistra-2.3$
+    decorate: true
+    path_alias: istio.io/istio
+    labels:
+      preset-quay-pusher: "true"
+    spec:
+      containers:
+      - command:
+        - entrypoint
+        - sh
+        - -c
```

Comment:
Remove sh and -c

Reply:
I think it's necessary so that `${PULL_NUMBER}` is resolved.